lldb/include/lldb/Target/Target.h

```diff
@@ -1446,7 +1453,9 @@ class Target : public std::enable_shared_from_this<Target>,
 
   // Runs the stop hooks that have been registered for this target.
   // Returns true if the stop hooks cause the target to resume.
-  bool RunStopHooks();
+  // Pass at_initial_stop if this is the stop where lldb gains
+  // control over the process for the first time.
+  bool RunStopHooks(bool at_initial_stop = false);
```

Comment:
Should this be `at_first_stop` for consistency with the target setting?

Reply:
I thought about this a bit, and I like `initial` better than `first`.  I want to make sure this isn't confused with "the first user stop", and `initial` sounds more like something that always happens.  So I switched everything over to initial.